service/src/test/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationServiceTest.java

```diff
@@ -78,6 +83,8 @@
 
     @BeforeEach
     void setUp() {
+        directionHelperService = new DirectionHelperService(userDetailsService);
```

Comment:
Sometimes you are doing this in `@beforeEach` and sometime on declaration, I think it would be better to be consistent